.env.development

```diff
@@ -14,13 +14,11 @@ REFRESH_ACCESS_TOKEN_ENDPOINT='http://localhost:18000/login_refresh'
 ACCESS_TOKEN_COOKIE_NAME='edx-jwt-cookie-header-payload'
 USER_INFO_COOKIE_NAME='edx-user-info'
 SITE_NAME=localhost
-
 DATA_API_BASE_URL='http://localhost:8000'
 // LMS_CLIENT_ID should match the lms DOT client application id your LMS containe
 LMS_CLIENT_ID='login-service-client-id'
-SEGMENT_KEY=null
+SEGMENT_KEY=''
 FEATURE_FLAGS={}
-CSRF_COOKIE_NAME='csrftoken'
```

Comment:
This variable was removed a few years ago and no longer exists.  Just pulled it out since I was here.